package.json

```diff
@@ -11,9 +11,9 @@
     "tsc": "tsc",
     "tsc:test": "tsc && npm run test --no-cache",
     "build:test": "npm run build && npm run test --no-cache",
-    "prebuild": "rm -rf dist && tsc",
+    "prebuild": "rimraf dist && tsc",
```

Comment:
yep, makes sense as on Windows you don't have `rm -rf`

Reply:
Actually, let's use similar approach here and have a prebuild script which calls `rimraf`. Node LTS offers recursive removal too but its still experimental https://nodejs.org/dist/latest-v12.x/docs/api/fs.html#fs_fs_rmdir_path_options_callback